### ISSUE TITLE
Functional Defect: Invisible Text on Password Screen in Non-Incognito Browser

### ISSUE THREAD

**Issue by ceciliabelem** [cambiatus/frontend#873]
**Describe the bug**
When accessing the password screen and pasting text into the edit box, the text appears in white, making it invisible to the user. This behavior occurs only in the non-incognito browser. In contrast, in incognito mode, the pasted text is displayed correctly in black on a white background.

**To Reproduce**

1. Open the browser in regular mode.
2. Access [Muda password's page](https://muda.cambiatus.io/login?redirect=%2Fdashboard)
3. Paste text into the edit box.
4. Observe that the pasted text appears in white, invisible to the user.

**Expected behavior**
Expected Behavior: The pasted text should be displayed in a visible color (black) in the edit box, allowing the user to see what has been entered.

**Current Behavior**
The pasted text appears in white, making it invisible in the edit box.

**Screenshots**
Non-Incognito Browser
![image](https://github.com/user-attachments/assets/165439e4-d6de-4a76-ad13-1897022a0b95)

Incognito Browser
![image](https://github.com/user-attachments/assets/1adbb5e4-34e3-4e7a-8225-1a42165404a3)

**Cambiatus Env you are using**
[Put the URL of the page you are trying to use](https://muda.cambiatus.io/login?redirect=%2Fdashboard)

**Desktop (please complete the following information):**
- OS: Windows
- Browser Chrome
- Version 129.0.6668.101 (Versão oficial) 64 bits

**Priority:** High

**Additional context**
